arches_lingo/src/arches_lingo/components/scheme/SchemeLabel/SchemeLabel.vue

```diff
@@ -0,0 +1,80 @@
+<script setup lang="ts">
+import { onMounted, ref } from "vue";
+
+import ProgressSpinner from "primevue/progressspinner";
+
+import SchemeLabelEditor from "@/arches_lingo/components/scheme/SchemeLabel/components/SchemeLabelEditor.vue";
+import SchemeLabelViewer from "@/arches_lingo/components/scheme/SchemeLabel/components/SchemeLabelViewer.vue";
+
+import { EDIT, VIEW } from "@/arches_lingo/constants.ts";
+
+import { fetchLingoResourcePartial } from "@/arches_lingo/api.ts";
+
+import type {
+    AppellativeStatus,
+    DataComponentMode,
+} from "@/arches_lingo/types.ts";
+
+const props = defineProps<{
+    mode: DataComponentMode;
+    tileId?: string | null;
```

Comment:
allowing both null and undefined is desired?

Reply:
yeah, not great I know. 

SchemeLabel in VIEW mode will not have a tileId prop. in EDIT mode the tileId is dependent on action. so we need the belt && suspenders here. 

( unless I"m misunderstanding the comment?)

Reply:
I guess I'm just wondering if we need to commingle two representation of emptiness. You prefer having null for EDIT mode versus undefined for VIEW? What about undefined for both?

Reply:
ah yeah I guess I"m saying I'd rather not pass an explicit null into the viewer eg

```
            <component
                :is="componentDatum.component"
                v-for="componentDatum in componentData"
                :key="componentDatum.componentName + '-' + componentDatum.key"
                :graph-slug="componentDatum.graphSlug"
                :nodegroup-alias="componentDatum.nodegroupAlias"
                :resource-instance-id="resourceInstanceId"
                :tile-id="null"
                :mode="VIEW"
            />
```

as it doesn't super make sense in that context, as it would _never_ have a tileid in view mode.

That said, I don't feel strongly about this one. Explicitness is almost never bad. /shrug

Reply:
Right, but if you choose `undefined` as the representation then you don't have to pass it in at all. I'm suggesting it's the editor that would set tileid.value = undefined instead of null for creates. This makes more sense from a REST sense too since on a python tile model tileid is not nullable, so you just won't send that k/v pair at all instead of sending "tileid: null".

Reply:
ah, there it is. yeah happy to make the change